vulcan/lib/client/jsx/components/workflow/user_interactions/inputs/multiple_string.tsx

```diff
@@ -0,0 +1,57 @@
+import React, {useState, useMemo, useEffect, useCallback, useRef} from 'react';
+import * as _ from 'lodash';
+
+import {InputBackendComponent} from './input_types';
+import TextInput from 'etna-js/components/inputs/text_input';
```

Comment:
I don't understand why this line is causing the vulcan tests to fail.  AFICT, this line is the same format as all the other etna-js component imports?

Reply:
Oh, you'll need to edit `etna.d.ts` and add a type definition for that component, if it hasn't been used before

Reply:
Basically giving typescript a hint for the parameter types and return format. There should be examples in there you can follow.